proknow/Patients/Entities/ImageSets.py

```diff
@@ -98,7 +98,7 @@ def get_image_data(self, index):
         """
         assert isinstance(index, int), "`index` is required as an integer."
         headers = {
-            "Accept-Version": "5",
+            "Accept-Version": self._rtv.get_api_version(type="imageset"),
```

Comment:
Not related to your changes, but it appears in the .NET SDK when I upgrade RTV to v3.2.2 that images are sent in random order and I cannot depend on the `index` pointing to the same image in the image set every time. 

If RTV is the cause, have you seen similar behavior with the Python SDK after upgrading to v3.2.2?

Reply:
I've not had a problem with the tests, but I think they're only using a single image.

I'm not sure the RTV Image Set API guarantees order. Perhaps you should sort `data.images` property by the position (`pos`)?

Reply:
Will do, wasn't sure if order should be guaranteed or not. You might also consider ordering images by position in this SDK as well if that's not already being done.